CHANGELOG.md

```diff
@@ -36,6 +36,11 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+## [v9.0.0-rc0] - 2023-01-19
+
+* (feat) Add [Interchain-Security](https://github.com/cosmos/interchain-security) [v1.0.0-rc2](https://github.com/cosmos/interchain-security/releases/tag/v1.0.0-rc2) provider module. See the [ICS Spec](https://github.com/cosmos/ibc/blob/main/spec/app/ics-028-cross-chain-validation/README.md) for more details.
+* (tests) Add [E2E ccv tests](https://github.com/cosmos/gaia/tree/main/tests/e2e). New tests covering functionality added by the provider module to add and remove a consumer chain via governance proposal.
```

Comment:
Hi, can you provide link to the 2e ccv test rather than e2e test?

Reply:
done